src/pages/komodo-defi-framework/api/v20/enable_eth_with_tokens/index.mdx

```diff
@@ -194,32 +203,137 @@ Additionally, it supports ERC20 tokens on the ETH chain and associated ERC20 lik
-  		"erc20_addresses_infos": {
-  			"0xaB95D01Bc8214E4D993043E8Ca1B68dB2c946498": {
-  				"derivation_method": {
-  					"type": "Iguana"
-  				},
-  				"pubkey": "04d8064eece4fa5c0f8dc0267f68cee9bdd527f9e88f3594a323428718c391ecc2a91c9ce32b6fc5489c49e33b688423b655177168afee1b128be9b2fee67e3f3b",
-  				"tickers": ["AAVE-PLG20", "PGX-PLG20"]
-  			}
-  		}
-  	},
-  	"id": null
+      "mmrpc": "2.0",
+      "result": {
+          "current_block": 60937035,
+          "eth_addresses_infos": {
+              "0x84ab…b27a": {
+                  "derivation_method": {
+                      "type": "Iguana"
+                  },
+                  "pubkey": "04ec603f83519cc2150bd99ed0ed6b3f7f029dedff2957cd22542e2504e3d2a7953c084f907cce2f0e26fb0cfc55e06925bf0f88a92b904224dba8ebafdf98ce7b"
+              }
+          },
+          "erc20_addresses_infos": {
+              "0x84ab…b27a": {
+                  "derivation_method": {
+                      "type": "Iguana"
+                  },
+                  "pubkey": "04ec603f83519cc2150bd99ed0ed6b3f7f029dedff2957cd22542e2504e3d2a7953c084f907cce2f0e26fb0cfc55e06925bf0f88a92b904224dba8ebafdf98ce7b",
+                  "tickers": [
+                      "PGX-PLG20",
+                      "AAVE-PLG20"
+                  ]
+              }
+          },
+          "nfts_infos": {}
+      },
+      "id": null
   }
   ```
 </CollapsibleSection>
 
+#### Request including NFT initialization
+
+<CodeGroup title="" tag="POST" label="enable_eth_with_tokens" mm2MethodDecorate="true">
+  ```json
+  {
+    "userpass": "RPC_UserP@SSW0RD",
+    "method": "enable_eth_with_tokens",
+    "mmrpc": "2.0",
+    "params": {
+      "ticker": "MATIC",
+      "get_balances": false,
+      "tx_history": false,
+      "nft_req": {
+        "type": "Moralis",
+        "info": {
+          "url": "https://moralis-proxy.komodo.earth"
+        }
+      },
+      "gas_station_url": "https://gasstation-mainnet.matic.network/",
+      "swap_contract_address": "0x9130b257D37A52E52F21054c4DA3450c72f595CE",
+      "fallback_swap_contract": "0x9130b257D37A52E52F21054c4DA3450c72f595CE",
+      "nodes": [
+        {
+          "url": "https://polygon-rpc.com"
+        },
+        {
+          "url": "https://node.komodo.earth:8080/polygon"
+        },
+        {
+          "url": "https://block-proxy.komodo.earth/rpc/matic"
+        }
+      ],
+      "erc20_tokens_requests": [
+        {
+          "ticker": "PGX-PLG20",
+          "required_confirmations": 4
+        },
+        {
+          "ticker": "AAVE-PLG20",
+          "required_confirmations": 4
+        }
+      ],
+      "required_confirmations": 5,
+      "requires_notarization": false
+    }
+  }
+  ```
+</CodeGroup>
+
+<CollapsibleSection expandedText="Hide Response" collapsedText="Show Response">
+  #### Response
+
+  ```json
+  {
+    "mmrpc": "2.0",
+    "result": {
+        "current_block": 61329162,
+        "eth_addresses_infos": {
+            "0xab95…6498": {
```

Comment:
Such eth address display was fixed  https://github.com/KomodoPlatform/komodo-defi-framework/pull/2254
Now response shows show full eth address format

please also fix it in other eth with tokens activation response examples

Reply:
fixed in https://github.com/KomodoPlatform/komodo-docs-mdx/pull/317/commits/ec9dad6493d8f7307267017f1273b66f146f0e13